packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -85,6 +88,32 @@
                 :data-test-id="`${testId.label}--inline`">
                 {{ labelText }}
             </form-label>
+
+            <span
+                v-if="hasLeftIcon"
```

Comment:
I would keep the naming to match the designs as much as possible.

So I'd change `left` > `leading` and `right` > `trailing` so that these would then become `hasLeadingIcon` and `hasTrailingIcon` (and the slots would then change to be `icon-leading` and `icon-trailing`) and the classnames the same as well.